### ISSUE TITLE
Training code & train using my own data

### ISSUE THREAD

**Issue by weijiawang96** [hoshino042/psnet#1]
Hi,

I am very interested in your work. From the code available (i.e. main.py), I found that it loads weights from a trained network model and directly tests on the input point clouds. However, I wish to retrain your network on my own dataset. Is the code for training available in this GitHub repository? If not, will you release it? I am really curious about how it'll look like after being trained on my own dataset.

Regards,
Wei

**Comment by weijiawang96** [comment]
Since the current code is for testing only, I really don't know how the loss functions work for the FELs and the decoding MLPs. For example, for all FEL layers, do we sum the loss for each layer, or only take the content from FEL-64 and take the style loss from FEL-2048? For the decoding MLPs, do we have a loss function to train them, or they are not trained at all? I am really curious about these points. Thanks in advance!
